packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -3194,6 +3194,7 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
 
   handleRollupChange(rollupConfig: UIRollupConfig): void {
     log.info('Rollup change', rollupConfig);
+    log.info('Rollup change str', JSON.stringify(rollupConfig));
```

Comment:
Did you still want the extra logging in this PR?

Reply:
Nope. D'oh